configure

```diff
@@ -23,13 +23,8 @@ function bazel_clean_and_fetch() {
   # TODO(pcloudy): Re-enable it after bazel clean --expunge is fixed.
   if ! is_windows; then
     bazel clean --expunge
-    # TODO(https://github.com/bazelbuild/bazel/issues/2220) Remove the nested `bazel query`.
-    bazel fetch $(bazel query "//tensorflow/... -//tensorflow/examples/android/...")
-  else
-    # TODO(pcloudy): Also filter out //tensorflow/examples/android/... on Windows after
-    # https://github.com/bazelbuild/bazel/issues/2248 is fixed.
-    bazel fetch //tensorflow/...
   fi
+  bazel fetch "//tensorflow/... -//tensorflow/examples/android/..."
```

Comment:
This change will work for now. The reason that I opted for the nested query instead of what you are proposing now is that _if_ someone from Bazel team gets around to fixing https://github.com/bazelbuild/bazel/issues/2220, I suspect that the fix for it would break this workaround.

And just to confirm, yes that failure is due to https://github.com/bazelbuild/bazel/commit/0b9ebfee8d7f3bd5cf68f4dba4790bf3fed9349d which I thought I resolved in #6225, but did not test on Windows. Sorry about that.